package.json

```diff
@@ -15,15 +15,21 @@
     "@chakra-ui/react": "^2.8.1",
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
+    "axios": "^1.6.8",
+    "body-parser": "^1.20.2",
     "cookie": "^0.6.0",
     "copy-to-clipboard": "^3.3.3",
+    "cors": "^2.8.5",
     "decode-uri-component": "^0.4.1",
+    "express": "^4.18.2",
```

Comment:
Would be surprised if you genuinely need this!

Reply:
ah yes, you're right i don't think we do